Tests/AESSpecTest.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
```

Comment:
Is this test added because AES was missing tests, or is there an update in the AES function of LNSym that needs these tests?